0.download_data/0.download_training_movies.sh

```diff
@@ -0,0 +1,2 @@
+jupyter nbconvert --to python aspera_training_downloader.ipynb
```

Comment:
```suggestion
#!/bin/bash
jupyter nbconvert --to python aspera_training_downloader.ipynb
```

You need a bash shebang (`#!`) right?